src/AWS/Lambda/Events/ApiGateway/ProxyRequest.hs

```diff
@@ -0,0 +1,167 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleInstances     #-}
+
+{-|
+Module      : AWS.Lambda.Events.ApiGateway.ProxyRequest
+Description : Data types that represent typical lambda responses
+Copyright   : (c) Nike, Inc., 2018
```

Comment:
I think generally in this PR we want to make this `2020`? :man_shrugging:

Reply:
The first commit just missed 2018, haha.  Will update these!

Reply:
Oh dang!